docs/howto/configure/pangeo-requester-pays-access.md

```diff
@@ -0,0 +1,103 @@
+# Pangeo Data Access via Requester Pays
+
+For some hubs, such as our Pangeo deployments, the communities they serve require access to data stored in other projects.
+Accessing data normally comes with a charge that the folks _hosting_ the data have to take care of.
+However, there is a method by which those making the request are responsible for the charges instead: [Requester Pays](https://cloud.google.com/storage/docs/requester-pays).
```

Comment:
Is "Requestor pays" access something unique to GCP? Or are we just prototyping this first on GCP?

Reply:
AFAIK, it is unique to GCP. I've not come across it before being introduced to the Pangeo deployment.